### PR TITLE
Added linux exercises

### PR DESCRIPTION
**Pull request by redfast00**

### CODE REVIEW

File: intro-Linux/ch_manipulating_files_and_directories.tex [hunk 1]
```diff
@@ -270,3 +270,20 @@ \subsection{Order of arguments}
 \begin{prompt}
     %\shellcmd{tar -c source1 source2 source3 -f tarfile.tar}%
 \end{prompt}
+
+\section{Exercises}
+
+\begin{enumerate}
+    \item Create a subdirectory in your home directory named \verb|test| containing
+        a single, empty file named \verb|one.txt|.
+    \item Copy \verb|/etc/hostname| into the \verb|test| directory and then check what's
```
[the remark this answers is not in the document]
Comment (redfast00) [reply intro-Linux/ch_manipulating_files_and_directories.tex]: Didn't see this comment for some reason :confused:.

[the remark this answers is not in the document]
Comment (boegel) [reply intro-Linux/ch_manipulating_files_and_directories.tex]: Not too late to fix ;)

[the remark this answers is not in the document]
Comment (redfast00) [reply intro-Linux/ch_manipulating_files_and_directories.tex]: That's  the thing: didn't see this comment, but did add the instruction (saw that you said the filename was `hostname.txt` and was confused, so I just put an instruction between).

[the remark this answers is not in the document]
Comment (boegel) [reply intro-Linux/ch_manipulating_files_and_directories.tex]: Ah, ok, good to go then.

**Pull request merged**